lib/UI/prev_workout/prev_workout_page.dart

```diff
@@ -83,7 +83,7 @@ class _PrevWorkoutPageState extends State<PrevWorkoutPage> {
                     Navigator.of(context).pop();
                     Navigator.of(context).push(
                       MaterialPageRoute(
-                        builder: (context) => CalendarPage(),
+                        builder: (context) => CalendarScreen(),
```

Comment:
:pray:

Reply:
We need to work on navigation. Do you have any idea about how many screens we have?

Reply:
Not a lot I think only 3? authentication screen, calendar screen and workout screen.

Reply:
how are we gonna handle navigation? Define named routes in the MaterialApp? or make our own navigator.